pennylane/gradients/parameter_shift.py

```diff
@@ -201,7 +201,7 @@ def processing_fn(results):
 
             # compute the linear combination of results and coefficients
             res = qml.math.stack(res)
-            g = sum([c * r for c, r in zip(gradient_coeffs[i], res)])
+            g = qml.math.tensordot(res, qml.math.convert_like(gradient_coeffs[i], res), [[0], [0]])

```

Comment:
I needed to change this because `sum` does not like Torch/TF :(

Reply:
:disappointed: